### DISCUSSION TITLE
Add placeholder "cmi_p_[playerDisplayName]_realname"

### DISCUSSION THREAD

**Discussion by Saros-6585** [Zrips/CMI#8831] [Feature Requests]
Display name can be obtained as `%cmi_p_[playerName]_display_name%`, but nickname(realname) can't.
There is `%cmi_user_name%`, but it is not suitable in some situations cause it must be parse.
so, please add `%cmi_p_[playerDisplayName]_realname%` (output is realname of player who matched with displayname)

**Comment by LucasHervieu** [comment]
use %cmi_user_display_name%

**Reply by Saros-6585** [reply]
You really understand my suggestion? That can't do my suggestion's goal.

**Reply by LucasHervieu** [reply]
yet it works with this variable for me
